common/service/apml/apml.h

```diff
@@ -142,6 +144,8 @@ void apml_init();
 void fatal_error_happened();
 void apml_recovery();
 int pal_check_sbrmi_command_code_length();
+uint8_t plat_set_apml_bus();
```

Comment:
Better to name this plat_get_apml_bus() since you are getting the bus from the platform rather than setting it.

Also shouldn't this go into plat_apml.h?

Reply:
Modify the function name to pal_get_apml_bus(), thank you.